src/mvContainers.cpp

```diff
@@ -70,6 +69,16 @@ DearPyGui::fill_configuration_dict(const mvChildWindowConfig& inConfig, PyObject
     checkbitset("menubar", ImGuiWindowFlags_MenuBar, inConfig.windowflags);
     checkbitset("no_scroll_with_mouse", ImGuiWindowFlags_NoScrollWithMouse, inConfig.windowflags);
     checkbitset("flattened_navigation", ImGuiWindowFlags_NavFlattened, inConfig.windowflags);
+
+    // child flags
+    checkbitset("border", ImGuiChildFlags_Border, inConfig.childFlags);
+    checkbitset("always_auto_resize", ImGuiChildFlags_AlwaysAutoResize, inConfig.childFlags);
+    checkbitset("always_use_window_padding", ImGuiChildFlags_AlwaysUseWindowPadding, inConfig.childFlags);
+    checkbitset("auto_resize_x", ImGuiChildFlags_AutoResizeX, inConfig.childFlags);
+    checkbitset("auto_resize_y", ImGuiChildFlags_AutoResizeY, inConfig.childFlags);
+    checkbitset("frame_style", ImGuiChildFlags_FrameStyle, inConfig.childFlags);
+    checkbitset("resize_x", ImGuiChildFlags_ResizeX, inConfig.childFlags);
```

Comment:
`resize_x`, `resize_y`: These have been renamed to `resizable_x/y` in `mvAppItem.cpp`.